pyuploadcare/api_resources.py

```diff
@@ -65,6 +66,19 @@ def __str__(self):
         return self.cdn_url
 
     @property
+    def uuid(self):
+        return self._uuid
+
+    @uuid.setter
+    def uuid(self, value):
+        match = RE_UUID_REGEX.match(value)
+
+        if not match:
+            raise InvalidRequestError('Invalid UUID: {0}'.format(value))
```

Comment:
Hm, you right, `InvalidRequestError` incorrect in this case but docstring says:

```python
class InvalidRequestError(UploadcareException, ValueError):
    """Invalid parameters errors, e.g. status 404."""
```

i.e. phrase `Invalid parameters errors` confusing me

And this exception also raises in `__init__` code where we just validate parameters without doing of request to service.

I propose to make a separate exception for all these cases e.g:

```python
class InvalidParamError(UploadcareException, ValueError):
   """Invalid parameters errors, e.g. invalid UUID"""

class InvalidRequestError(InvalidParam):
   """Invalid service parameters errors, e.g status 404"""
```

What do you think about this?